networking/init-cms-bundle/4.0/config/packages/networking_init_cms.yaml

```diff
@@ -0,0 +1,101 @@
+imports:
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/doctrine.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/fos_ck_editor.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/fos_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/mopa_bootstrap.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/oneup_flysystem.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/oneup_uploader.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_block.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_core.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_doctrine_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_formatter.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_media.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_notification.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/stof_doctrine_extensions.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/symfony_cmf_routing_extra.yaml" }
+# Default security configuration
+#    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/security.yaml" }
+
+# Set templating to twig. FOSUserBundle requires templating to be setup
+framework:
+    templating:      { engines: ['twig'] }
+    cache:
+        pools:
+            page.cache:
+                adapter: cache.app #Setup page cache using template from cache.app
+
+# Set up the serializer to read config file for App Page and User entities
+jms_serializer:
+    metadata:
+        directories:
+            NetworkingSonataMediaBundle:
+                namespace_prefix: "Sonata\\MediaBundle"
+                path: "@NetworkingInitCmsBundle/Resources/config/serializer"
+            app:
+                namespace_prefix: "App"
+                path: "%kernel.project_dir%/config/serializer"
+
+# Translation bundle configuration, swap environment variables if mange_locales_only option set to false
+lexik_translation:
```

Comment:
Please remove

Reply:
The same thing here applies, if there are no default locale set the pages can not be created when running the install command for the cms. Unfortunately many of the bundles that we use do not yet have a recipe. We want to able for a user to get the CMS up and running with as little configuration and file copying as possible.

Also, our company is one of the only group of people who use this CMS. It is developed for us to quickly set up a CMS for our customers.

Reply:
There are some issues will modifying other bundles configuration. 

- But what if someone got this bundle already installed? Then you have two places where you defined `lexik_translation`'s default locale
-LexikTranslation bundle's recipe will also run. This will create multiple configurations for the same bundle.

Reply:
There is not LexikTranslationBundle recipe, that is exactly the point.

Reply:
Also, as mentioned in our post-instal.txt we do point out that there may be cross over in the configuration, which is why we state that the user should check first before proceeding.

That is mentioned in the very first point.

```
    1. Check that all files imported via the networking_init_cms.yaml file are not overwritten with a similar
       configuration file in your /config/packages folder. !IMPORTANT the doctrine.yaml and security.yaml must be either
       deleted, or update/extended with the configuration as proposed in the doctrine.yaml and security.yaml files
       included with the CMS.
```